Formula/libeatmydata.rb

```diff
@@ -0,0 +1,42 @@
+class Libeatmydata < Formula
+  desc "LD_PRELOAD library and wrapper to transparently disable fsync and related calls"
+  homepage "https://www.flamingspork.com/projects/libeatmydata/"
+  url "https://github.com/stewartsmith/libeatmydata/releases/download/v130/libeatmydata-130.tar.gz"
+  sha256 "48731cd7e612ff73fd6339378fbbff38dd3bcf6c243593b0d9773ca0051541c0"
+  license "GPL-3.0-or-later"
+
+  head do
+    url "https://github.com/stewartsmith/libeatmydata.git", branch: "master"
+    depends_on "autoconf"         => :build
+    depends_on "autoconf-archive" => :build
+    depends_on "automake"         => :build
+    depends_on "libtool"          => :build
+  end
+
+  depends_on "coreutils"
+
+  on_linux do
+    depends_on "strace" => :test
+  end
+
+  def install
+    # macOS does not support `readlink -f` as used by the `eatmydata` shell wrapper script
+    inreplace "eatmydata.sh.in", "readlink", "#{Formula["coreutils"].opt_libexec}/gnubin/readlink"
```

Comment:
```suggestion
    inreplace "eatmydata.sh.in", "readlink", "#{Formula["coreutils"].opt_bin}/greadlink"
```
Not sure this works on Linux though. Also, is there no way to do
```ruby
ENV["READLINK"] = "greadlink"
```
or similar?

Why didn't we need this when we did `autoreconf`?

Reply:
It's actually the wrapper script itself - I had always had coreutils there since the original test - with it removed, the failure is https://github.com/Homebrew/homebrew-core/pull/106536/commits/17b6dd2c0ae300792500a1d4eaa3e355de4c8c23

Reply:
The problematic shell section is:

![image](https://user-images.githubusercontent.com/61629094/181358923-b9dbc66c-8354-4a40-95f4-0f46a29a1807.jpeg)

I can only make the patch on macOS - it's probably very safe to assume that any Linux system will have GNU userland and coreutils already.

The exceptions might be Busybox-based systems - if so, a Linux patch could be made too, but probably unnecessary.

I'll report this upstream.

Reply:
Should be OK in https://github.com/Homebrew/homebrew-core/pull/106536/commits/56a8f8be4000c2aa52384272321939cdf9760987